Dockerfile

```diff
@@ -2,11 +2,27 @@ FROM python:3.6-jessie
 
 RUN useradd -ms /bin/bash ark
 
+ADD . /usr/src/code
+WORKDIR /usr/src/code
+
 ENV DJANGO_SETTINGS_MODULE=app.settings \
-    PYTHONPATH=/usr/src/code/src
+    PYTHONPATH=/usr/src/code/src \
+    DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y postgresql
+    apt-get install -y postgresql && \
+    apt-get install -y apt-transport-https && \
+    apt-get install -y build-essential && \
+    apt-get install gcc g++ make
+
+# install nodejs
+RUN apt-get clean && \
+    curl -sL https://deb.nodesource.com/setup_8.x | bash
+
+# install yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
```

Comment:
Question: What's the benefit of using yarn vs npm nowadays?

Reply:
it used to be speed, but I know that npm improved since. I'm not entirely aware what are benefits nowadays but it seems to be more and more of a standard over npm. 🤷‍♂️

Reply:
I know the reason why we moved to yarn was because of yarn.lock file. But now npm also supports that with their version 2.0 or whatever it's called. I haven't looked into it, but would probably be neat to check what the benefits are and remove yarn if possible just to not have another build step in dockerfile. It looks like this adds a lot of bloat to it.